README.md

```diff
@@ -192,14 +192,9 @@ Voice + youtube-dl:
 
 - youtube-dl (Arch: `community/youtube-dl`)
 
-# Related Projects
+# Projects extending Serenity
 
-- [Discord.net][library:Discord.net]
-- [JDA][library:JDA]
-- [disco][library:disco]
-- [discordrb][library:discordrb]
-- [discord.js][library:discord.js]
-- [discord.py][library:discord.py]
+- [serenity-lavalink][project:serenity-lavalink] - A wrapper around [Lavalink][repo:lavalink], an audio sending node based on [Lavaplayer][repo:lavaplayer]
```

Comment:
I believe it would be better to describe it as an interface to Lavalink. Lavalink is a stand alone server so the only way to interface with it it through it's network API.